tests/sentry/api/endpoints/test_organization_releases.py

```diff
@@ -2101,6 +2101,61 @@ def test_bad_repo_name(self) -> None:
         assert response.status_code == 400
         assert response.data == {"refs": ["Invalid repository names: not_a_repo"]}
 
+    def test_user_auth_token_with_project_releases_scope_non_member(self) -> None:
+        """
+        Test that a user with a token that has project:releases scope can create
+        releases for projects they're not a team member of, ensuring consistency
+        with the project releases endpoint behavior.
+        """
+        user = self.create_user(is_staff=False, is_superuser=False)
+        org = self.create_organization()
+        org.flags.allow_joinleave = False
+        org.save()
+
+        team1 = self.create_team(organization=org)
+        team2 = self.create_team(organization=org)
+
+        # User is only a member of team1
```

Comment:
This comment is confusing because at this point the user hasn't been added to any team yet.
```suggestion
```